libgalois/test/barriers.cpp

```diff
@@ -77,7 +77,7 @@ test(std::unique_ptr<katana::Barrier> b) {
 
 int
 main(int argc, char** argv) {
-  katana::SharedMemSys Katana_runtime;
+  katana::GaloisRT Katana_runtime;
```

Comment:
I don't think `GaloisRT` is any better than `GaloisSys` or even `GaloisRuntime`

Reply:
Agree. Will change to `GaloisRuntime`.

Reply:
Why change in name? SharedMemSys and DistMemSys seemed reasonable to me

Reply:
So, we still have `SharedMemSys` in the new `libgraph` because, now, `SharedMemSys` initializes `libtsuba` and `libgalois`